### PR TITLE
Add a generic description to the alg

### PR DESCRIPTION
**Pull request by DelazJ**
same as what is available in the GUI
Move the Z-score output explanation to where it's mentioned (refs https://github.com/qgis/QGIS-Documentation/pull/5963#issuecomment-669513724)

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/vectoranalysis.rst [hunk 1]
```diff
@@ -1067,15 +1067,8 @@ Python code
...
-with the normal distribution tells you how your data are distributed.
-
-A low Z-Score means that the data are unlikely distributed randomly
-while a high Z-Score means that your data are very likely
-distributed in a dispersed way.
-
-.. figure:: img/normal_distribution.png
-  :align: center
+Performs nearest neighbor analysis for a point layer. The output allows to
+understand how your data are distributed (clustered, randomly or distributed).
```
Comment (havatv) [inline docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1071]:
```suggestion
Performs nearest neighbor analysis for a point layer. The output tells you
how your data are distributed (clustered, randomly or distributed).
```

File: docs/user_manual/processing_algs/qgis/vectoranalysis.rst [hunk 2]
```diff
@@ -1084,7 +1077,15 @@ values:
 * Expected mean distance
 * Nearest neighbour index
 * Number of points
-* Z-Score
+* Z-Score: Comparing the Z-Score with the normal distribution tells
+  you how your data are distributed.
+  A low Z-Score means that the data are unlikely distributed randomly
+  while a high Z-Score means that your data are very likely
+  distributed in a dispersed way.
```
Comment (havatv) [inline docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1084]:
```suggestion
  A low Z-Score means that the data are unlikely to be the result of a
  spatially random process, while a high Z-Score means that your data
  are likely to be a result of spatially random process.
```

Comment (DelazJ) [reply docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1084]: **are** (un)likely **to be**?

Comment (havatv) [reply docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1084]: > **are** (un)likely **to be**?

?

**Pull request merged**